ydb/public/api/protos/ydb_topic.proto

```diff
@@ -796,24 +811,89 @@ message AlterConsumer {
     map<string, string> alter_attributes = 6;
 }
 
+enum AutoScalingStrategy {
+    // Auto scaling disabled
+    DISABLED = 0;
```

Comment:
what about reserve 0 for `Undefined` of `Default` - for we can change default behaviour in the future on server side?

What about move `AutoScalingStrategy` into message `AutoScalingSettings`?